### ISSUE TITLE
V6.0 reports errors when V7.20 does not on Intel 13th and 14th gen

### ISSUE THREAD

**Issue by pawelm8** [memtest86plus/memtest86plus#471]
Hello,
I was using MemTest86+ version 6.0 (included with Parted Magic) and it found some errors with a couple of my machines, but then when testing with the latest 7.20 version the errors were gone. This was happening with Intel 13th and 14the gen CPUs and I would assume some incompatibility with the older version, but more tests suggests that might not be the case.
The v6.0 tests were always failing consistently around 70% mark with Test # 9 "Modulo 20, random pattern".

Here are the details:
1) 14700 (non-K) tested on two Gigabyte boards (Z790 ATX and B760 mATX) with different DDR5 memory kits (6000 CL32 and 7200 CL34, with and without XMP enabled) was showing errors with the version 6.0, while a different CPU 12900F showed no errors in all scenarios. I assumed the CPU was faulty and returned it.
2) later I've run the v6.0 test on my laptops: Asus Strix Scar 18 (2023) with i9-13980HX showed errors with 3 different memory kits (DDR5-4800, DDR5-5600 and Kingston DDR5-5600 CL40), same as Asus Strix G18 (2024) with i9-14900HX (also with different memory kits)
3) another laptop Dell XPS 9730 with i9-13900H showed no errors with v6.0
4) also a brand new 14600K installed in the same B760 setup with DDR5-7200 memory (as tested in point 1 with 14700) showed no errors with v6.0
5) but the same laptops mentioned in point 2 shows no errors when tested with version 7.20.

In short: version 7.20 shows no errors in every setup, while version 6.0 shows errors in some cases and replacing the CPU (a new 14600K instead of a used 14700) seems to help get rid of the errors.

How should I understand such results? Is v6.0 buggy when used with some CPUs?
Both 13900H and 14600K are basically the same spec with 6P and 8E cores, while the problematic CPUs are all 8 P-cores and 12 or 16 E-cores. But then 12900F is also an 8 P-cores CPU, so it's hard to see the pattern here.
I appreciate any help here.

**Comment by debrouxl** [comment]
Hello, thanks for the report. Looks like another occurrence of #216 , a known bug in memtest86+ 6.00 indeed.

**Comment by debrouxl** [comment]
Yup, that's the information I asked for :)
It shows that there's a single 4K page range in that list (9f-a0). Therefore, I'm pretty sure that this is another occurrence of #216 , which only affects memtest86+ 6.00, and is not necessarily easy to find by searching in this bug tracker.

**Comment by pawelm8** [comment]
Thank you for the confirmation :).
I've got the same 14700 back (which I initially assumed is faulty) and, as expected, it produces no errors when tested with 7.20.

I'll close this issue then. Thanks again!

**Comment by debrouxl** [comment]
You're welcome :)
